adapters/http4s/src/main/scala/caliban/Http4sAdapter.scala

```diff
@@ -208,30 +207,30 @@ object Http4sAdapter {
                     }
                   case "stop" =>
                     val id = msg.hcursor.downField("id").success.flatMap(_.value.asString).getOrElse("")
-                    subscriptions
```

Comment:
I think that part is needed. When we receive an `unsubscribe` request (`stop`), we want to interrupt the fiber that was consuming the stream for that ID (and keep the other fibers running). 
We no longer need to use `subscriptions` for the keep-alive fiber but I think it's still needed for each subscription so that we can stop them before the WebSocket is closed.

Reply:
You're right! I'll fix this.